fedora/client/openidbaseclient.py

```diff
@@ -182,7 +183,8 @@ def _authed_get(self, url, params=None, data=None, **kwargs):
         response = self._session.get(url, params=params, data=data, **kwargs)
         return response
 
-    def send_request(self, method, auth=False, verb='POST', **kwargs):
+    def send_request(self, method, auth=False, verb='POST', retries=None,
+                     retry_sleep=0.5, **kwargs):
```

Comment:
On Wed, Sep 30, 2015 at 08:40:49AM -0700, Kamil Páral wrote:
>    In fedora/client/openidbaseclient.py:
> 
>  > @@ -182,7 +183,8 @@ def _authed_get(self, url, params=None, data=None, **kwargs):
>  >          response = self._session.get(url, params=params, data=data, **kwargs)
>  >          return response
>  > 
>  > -    def send_request(self, method, auth=False, verb='POST', **kwargs):
>  > +    def send_request(self, method, auth=False, verb='POST', retries=None,
>  > +                     retry_sleep=0.5, **kwargs):
> 
>    OK, so now you should kick it out of the send_request() parameters and use
>    self.retry_sleep inside it instead.

Retries is present in both place on the old client, why not keep it here as
well?